composeApp/src/commonMain/kotlin/sp/bvantur/tasky/event/presentation/CreateEventViewState.kt

```diff
@@ -8,5 +9,19 @@ import tasky.composeapp.generated.resources.event_title
 
 data class CreateEventViewState(
     val title: TextData = TextData.ResourceString(Res.string.event_title),
-    val description: TextData = TextData.ResourceString(Res.string.event_description)
+    val description: TextData = TextData.ResourceString(Res.string.event_description),
+    val showDatePickerDialog: Boolean = false,
+    val showTimePickerDialog: Boolean = false,
+    val dialogDateTimeData: DialogDateTimeData? = null,
+    val dialogSelectedTimeInMillis: Long = 0L,
+    val currentFromDateTime: LocalDateTime? = null,
+    val currentToDateTime: LocalDateTime? = null,
+    val currentFromInMillis: Long = 0L,
+    val currentToInMillis: Long = 0L,
+    val formattedFromDate: String = "",
+    val formattedFromTime: String = "",
+    val formattedToDate: String = "",
+    val formattedToTime: String = "",
```

Comment:
In my experience, this approach is very error-prone with hard-to-find bugs that can come up just because you updated one of the fields (for example, `currentToInMillis`) and not another (for example, `currentToDateTime`)

Could it not make more sense to just have a single field for the to and from times instead of representing the same state in different formats?

If you need just the time or date you can have extension functions such as this:

```
fun LocalDateTime.formatAsTime(): String {
    val formatter = DateTimeFormatter.ofPattern("HH:mm")
    return this.format(formatter)
}

fun LocalDateTime.formatAsDate(): String {
    val formatter = DateTimeFormatter.ofPattern("MMM dd yyyy")
    return this.format(formatter)
}
```